Bottom: a3d5e444c7f1040aa77e98acdb35f9ad018562ce
Top:    22dcf0fcd66e5aa67838dcb28f112583010fb1ef
Author: Anton Dubniak <twister@tfsoft.org.ua>
Date:   2015-12-30 17:53:00 +0200

From cfcb31d533fd723c238870abe2f8f3a5b1cc5000 Mon Sep 17 00:00:00 2001
Subject: [PATCH] A possible fix for the LP1013358 bug


---

diff --git a/src/crosshair.c b/src/crosshair.c
index fb2ceca..3999944 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -591,7 +591,7 @@ DrawAttached (hidGC gc)
 	  if (TEST_FLAG (SHOWDRCFLAG, PCB))
 	    {
 	      gui->graphics->set_color (gc, Settings.CrossColor);
-	      XORDrawAttachedArc (gc, Settings.LineThickness + 2 * (PCB->Bloat + 1));
+	      XORDrawAttachedArc (gc, Settings.LineThickness + 2 * PCB->Bloat);
 	      gui->graphics->set_color (gc, Settings.CrosshairColor);
 	    }
 
@@ -622,13 +622,13 @@ DrawAttached (hidGC gc)
 	                           Crosshair.AttachedLine.Point2.X,
 	                           Crosshair.AttachedLine.Point2.Y,
 	                           PCB->RatDraw ? 10 : Settings.LineThickness
-	                           + 2 * (PCB->Bloat + 1));
+	                           + 2 * PCB->Bloat);
 	      if (PCB->Clipping)
 		XORDrawAttachedLine (gc, Crosshair.AttachedLine.Point2.X,
 		                         Crosshair.AttachedLine.Point2.Y,
 		                     Crosshair.X, Crosshair.Y,
 		                     PCB->RatDraw ? 10 : Settings.
-		                     LineThickness + 2 * (PCB->Bloat + 1));
+		                     LineThickness + 2 * PCB->Bloat);
 	      gui->graphics->set_color (gc, Settings.CrosshairColor);
 	    }
 	}
diff --git a/src/line.c b/src/line.c
index 469f0f6..6c90a3c 100644
--- a/src/line.c
+++ b/src/line.c
@@ -304,7 +304,7 @@ drc_lines (PointType *end, bool way)
   s = 0.5;
   last = -1;
   line1.Flags = line2.Flags = NoFlags ();
-  line1.Thickness = Settings.LineThickness + 2 * (PCB->Bloat + 1);
+  line1.Thickness = Settings.LineThickness + 2 * PCB->Bloat;
   line2.Thickness = line1.Thickness;
   line1.Clearance = line2.Clearance = 0;
   line1.Point1.X = Crosshair.AttachedLine.Point1.X;
